docs/installation/linux_sources.rst

```diff
@@ -60,3 +60,62 @@ To install eProsima Shapes Demo using colcon, please follow the steps below:
     .. code-block:: bash
 
         ShapesDemo
+
```

Comment:
A general comment for this page: In Jammy, it is possible to install Qt5 with apt; maybe we could add a note or something with the instructions.

Reply:
Maybe we can move the sentence about installing Qt 5 from the `ROS 2 features compilation` section to the first paragraph of the page